src/Components/CreateImageWizard/steps/Registration/components/SatelliteRegistrationCommand.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+
+import {
+  FormGroup,
+  FormHelperText,
+  HelperText,
+  HelperTextItem,
+} from '@patternfly/react-core';
+
+import { useAppDispatch, useAppSelector } from '../../../../../store/hooks';
+import {
+  changeSatelliteRegistrationCommand,
+  selectSatelliteRegistrationCommand,
+} from '../../../../../store/wizardSlice';
+import { useRegistrationValidation } from '../../../utilities/useValidation';
+import { HookValidatedInput } from '../../../ValidatedInput';
+
+const SatelliteRegistrationCommand = () => {
+  const dispatch = useAppDispatch();
+  const registrationCommand = useAppSelector(
+    selectSatelliteRegistrationCommand
+  );
+  const stepValidation = useRegistrationValidation();
+
+  const handleChange = (e: React.FormEvent, value: string) => {
+    dispatch(changeSatelliteRegistrationCommand(value));
+  };
+
+  return (
+    <FormGroup label="Registration command from Satellite" isRequired>
+      <HookValidatedInput
+        inputType={'textArea'}
+        ariaLabel="registration command"
+        value={registrationCommand || ''}
+        onChange={handleChange}
+        placeholder="Registration command"
+        stepValidation={stepValidation}
+        fieldName="command"
+      />
+      <FormHelperText>
+        <HelperText>
+          {/* TODO: Add link to our docs */}
+          <HelperTextItem>
+            To generate command from Satellite, follow the documentation.
```

Comment:
Should we link the current Satellite registration for now? :)
https://docs.redhat.com/en/documentation/red_hat_satellite/6.16/html-single/managing_hosts/index#Customizing_the_Registration_Templates_managing-hosts

Reply:
We could, both no link and using a temporary relevant link work for me. We just need to remember to open a follow up when the new documentation is ready. :+1: